config/standalone/CHIPProjectConfig.h

```diff
@@ -85,6 +85,6 @@
 //
 #define CHIP_CONFIG_MAX_EXCHANGE_CONTEXTS 150
 
-#define CONFIG_IM_BUILD_FOR_UNIT_TEST 1
+#define CONFIG_BUILD_FOR_UNIT_TEST 1
```

Comment:
I think it's worthwhile adding a comment here that this flag is not going to be enabled on devices, so if you have a test that uses this flag, to either appropriately conditionalize the entire test on this flag, or to exclude the compliation of that test source file entirely.

Reply:
Should the comment be here or in `src/lib/core/CHIPConfig.h`?

Reply:
Also should we go with `CONFIG_BUILD_FOR_HOST_UNIT_TEST`?